lrlex/src/lib/mod.rs

```diff
@@ -27,15 +27,16 @@ pub use crate::{
     parser::StartState,
 };
 
+use cfgrammar::yacc::parser::SpansKind;
```

Comment:
Should we export this from top-level, so we don't have to reach into parser from within lrlex?

Reply:
Not quite sure I understand?

Reply:
Just that it is reusing this type which is declared within `cfgrammar::yacc`, from within the `lrlex` crate,
rather than a neutral place like `cfgrammar::span::SpanKind` where it can be used by *both* `cfgrammar::yacc` and `lrlex`, but keeping it where it is (close to the errors from which it derives its meaning might be preferred organizationally).

I think it is harmless, but dependency-wise I wasn't certain it made sense to actually depend upon the `yacc` module here.

Reply:
Oh, I'm with you! So make this a `pub use` in `cfgrammar/mod.rs`?

Reply:
Yeah, that is what I was thinking!

Reply:
Having double-checked this, I think the top-level of cfgrammar should be somewhat independent of yacc, and `SpansKind` is (at least at the moment) seemingly yacc specific, so I think `use cfgrammar::yacc::parser::SpansKind;`, while long, is the right way to do things.

Reply:
Alright, I'll leave this as-is then.